contracts/interfaces/IAssessment.sol

```diff
@@ -61,15 +39,13 @@ interface IAssessment is IERC721Receiver {
    */
   struct Vote {
    // Identifier of the claim or incident
-    uint104 eventId;
+    uint80 assessmentId;
    // If the assessor votes to accept the event it's true otherwise it's false
     bool accepted;
    // Date and time when the vote was cast
     uint32 timestamp;
    // How many tokens were staked when the vote was cast
     uint96 tokenWeight;
```

Comment:
'weight' is misleading here. alternative suggestions:
- `stakeAmount`
- `tokensStaked`